src/plots/cartesian/axes.js

```diff
@@ -1799,12 +1811,18 @@ function numFormat(v, ax, fmtoverride, hover) {
         // 0 is just 0, but may get exponent if it's the last tick
         v = '0';
         isNeg = false;
-    }
-    else {
+    } else {
```

Comment:
sometimes I drop an extra newline right before the batwing `} else {` just so the break is easy to identify visually. Just so long as things don't get too scrunchy!  Also we have a JS styleguide with such things to resolve all matters of preference. 
https://github.com/plotly/dev-docs/blob/master/reference/styleguides/javascript.md